README.md

```diff
@@ -0,0 +1,15 @@
+## Ravelin Data Science Test
+
+Below are some json profiles representing fictional customers from an ecommerce company. The profiles contain information about the customer, their orders, their transactions, what payment methods they used and whether the customer is fraudulent or not. Your task is one of the following.
+
+### Option A
+ * Transform the json profiles into feature vectors 
+ * Construct a model to predict if a customer is fraudulent based on their profile.
+ * Report on the models success and show what features are most important
+
+### Option B
+ * Provide exploratory analysis of the dataset.
```

Comment:
Maybe just me but I would have freaked out if I was told to do EDA on a json file, might be worth including an extra step here saying convert to csv or pandas dataframe then do EDA?